build failed about atomic

hi 
just encounter an issue with latest commit 6f9a9a93b6d8355f97f3073b31006119faf81f21

> paddle/framework/operator.cc: In member function ‘void paddle::framework::OperatorBase::GenerateTemporaryNames()’:
> paddle/framework/operator.cc:194:10: error: ‘atomic’ in namespace ‘std’ does not name a type
>    static std::atomic<size_t> gUniqId(0UL);

CentOS Linux release 7.3.1611 (Core)

gcc version
> g++ (GCC) 4.8.5 20150623 (Red Hat 4.8.5-11)
> Copyright (C) 2015 Free Software Foundation, Inc.


It works after added `#include <atomic>`

--- a/paddle/framework/operator.cc
+++ b/paddle/framework/operator.cc
@@ -14,6 +14,7 @@ limitations under the License. */
 
 #include "paddle/framework/operator.h"
 #include <algorithm>
+#include <atomic>
 
 namespace paddle {
 namespace framework {

